### PR TITLE
MAP-17 Location Type Component

### PR DESCRIPTION
**Pull request by acazacu**
- Removed the ability to zoom in on the app from mobile devices.
- Changedthe tablet portrait breakpoint to allow for mobile landscape devices to function in the mobile layout.
- Implemented the LocationType Component with unit test coverage.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,user-scalable=no">
```
Comment (janjaap) [inline public/index.html]: Very user unfriendly. Was this a specific client request?
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#Viewport_scaling for reference

File: src/assets/carret-down.svg [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (janjaap) [inline src/assets/carret-down.svg]: You can compress this SVG. [Here](https://jakearchibald.github.io/svgomg/), for instance.
Same applies to the other SVGs in this PR

File: src/components/inputs/LocationType.vue [hunk 3]
```diff
@@ -0,0 +1,200 @@
+<template>
+  <div ref="rootEl">
+    <button :class="selectedClass" @click="onTriggerClick"></button>
+    <transition name="fade">
+      <ul v-if="isListVisible">
+        <li v-for="option in options" v-bind:key="option.value">
+          <a :class="option.key" alt="option.description" @click="onListItemClick(option.value)">{{
```
Comment (janjaap) [inline src/components/inputs/LocationType.vue]: Should an anchor tag have an `alt` attribute?

File: src/components/inputs/LocationType.vue [hunk 4]
```diff
@@ -0,0 +1,200 @@
+<template>
+  <div ref="rootEl">
+    <button :class="selectedClass" @click="onTriggerClick"></button>
```
Comment (janjaap) [inline src/components/inputs/LocationType.vue]: Can be a self-closing element

File: src/components/inputs/LocationType.vue [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+    position: absolute;
+    top: 100%;
+    left: 0;
+    margin: 16px 0 0 0;
+    background: white;
+    list-style: none outside;
+    width: 162px;
+    padding: 9px 12px;
+    border-radius: 3px;
+    box-shadow: 0 2px 4px 0 #00000055;
```
Comment (janjaap) [inline src/components/inputs/LocationType.vue]: Alpha hex-color notation is not supported in IE. Is this value handled by the post processor?

File: src/components/inputs/LocationType.vue [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+    margin: 5px 11px;
+
+    a {
+      display: block;
+      width: 30px;
+      padding-top: 30px;
+      background-size: 26px 26px;
+      background-position: center 2px;
+      background-repeat: no-repeat;
+      cursor: pointer;
```
Comment (janjaap) [inline src/components/inputs/LocationType.vue]: Isn't this the default value?

File: public/index.html (hunk 1 above)
Comment (acazacu) [reply public/index.html]: Not really, it's a hack to disable the viewport automatically zooming in on focused inputs and selects. As I understand it, there is a 16px font-size threshold for that to NOT happen on iOS devices. My native select box uses a font size of 0 to completely hide its content and replace it with a background image... If you know of any other solutions to this, I'd be happy to remove that user-scalable property right there :)

File: src/components/inputs/LocationType.vue (hunk 3 above)
Comment (acazacu) [reply src/components/inputs/LocationType.vue]: You are right, it was title that I was looking for. Good spot!

File: src/components/inputs/LocationType.vue (hunk 5 above)
Comment (acazacu) [reply src/components/inputs/LocationType.vue]: Actually, this is supposed to be a variable and I forgot to make it so. Fixing the alpha notation with it as well.

File: src/components/inputs/LocationType.vue (hunk 6 above)
Comment (acazacu) [reply src/components/inputs/LocationType.vue]: Not without an href tag, so far that I know. The cursor would remain in its default state...

File: src/assets/carret-down.svg (hunk 2 above)
Comment (acazacu) [reply src/assets/carret-down.svg]: Hey this is a nice tool! I think it slimmed down the payloads by about 50%. I'm going to look for a way to have this integrated into the build step, done automatically. Thank you!

File: public/index.html (hunk 1 above)
Comment (acazacu) [reply public/index.html]: alright, I found another fix for this that did not have me hacking the viewport. Resolved!

File: src/components/inputs/LocationType.vue (hunk 6 above)
Comment (acazacu) [reply src/components/inputs/LocationType.vue]: Replaced with a button tag as discussed, it makes more sense

**Pull request merged**